lib/Conversion/TorchToTosa/TorchToTosa.cpp

```diff
@@ -3994,6 +3996,8 @@ class ConvertTorchToTosa : public ConvertTorchToTosaBase<ConvertTorchToTosa> {
     INSERT_BINARY_COMPARE_PATTERN(AtenNeTensorOp, tosa::EqualOp)
     INSERT_BINARY_COMPARE_PATTERN(AtenNeScalarOp, tosa::EqualOp)
     INSERT_BINARY_COMPARE_PATTERN(AtenBitwiseAndTensorOp, tosa::BitwiseAndOp)
+    INSERT_BINARY_COMPARE_PATTERN(AtenBitwiseOrTensorOp, tosa::BitwiseOrOp)
+    INSERT_BINARY_COMPARE_PATTERN(AtenBitwiseXorTensorOp, tosa::BitwiseXorOp)
```

Comment:
Can you add an e2e test that passes in the TOSA backend?

Reply:
Okay, I added two e2e tests for each bitwise binary op, one with dynamic shape and the other with static shape. After fixing type promotion issue, now TOSA can pass both dynamic & static shape tests, and MHLO passes only the static one.